### PR TITLE
Suisin/Create dob and name mismatch banner

### PR DESCRIPTION
**Pull request by suisin-deriv**

### CODE REVIEW

File: src/javascript/app/base/header.js [hunk 1]
```diff
@@ -361,10 +361,13 @@ const Header = (() => {
             };
 
             const messages = {
-                cashier_locked    : () => localize('Your cashier is currently locked. Please contact us via live chat to find out how to unlock it.'),
-                system_maintenance: () => getSystemMaintenanceMessage(),
-                currency          : () => buildMessage(localizeKeepPlaceholders('Please set your [_1]account currency[_2] to enable deposits and withdrawals.'),                                                                                    'user/set-currency'),
-                unsubmitted       : () => buildMessage(get_account_status.risk_classification === 'high'
+                cashier_locked        : () => localize('Your cashier is currently locked. Please contact us via live chat to find out how to unlock it.'),
+                system_maintenance    : () => getSystemMaintenanceMessage(),
+                currency              : () => buildMessage(localizeKeepPlaceholders('Please set your [_1]account currency[_2] to enable deposits and withdrawals.'),                                                                                    'user/set-currency'),
```
Comment (mahdiyeh-fs) [inline src/javascript/app/base/header.js:366]: Please remove these unnecessary spaces between message and link.

Comment (suisin-deriv) [reply src/javascript/app/base/header.js:366]: Orite will do so 🙏

**Review by mahdiyeh-fs — APPROVED** [review]

**Comment by mahdiyeh-fs** [comment]
LGTM +1

**Review by yauheni-kryzhyk-deriv — APPROVED** [review]

**Comment by yauheni-kryzhyk-deriv** [comment]
LGTM

File: src/javascript/app/base/header.js (hunk 1 above)
Comment (matin-binary) [reply src/javascript/app/base/header.js]: Lol

**Review by matin-binary — APPROVED** [review]

**Review by prince-deriv — APPROVED** [review]

**Pull request merged**